app/views/facilities/_form.html.erb

```diff
@@ -0,0 +1,21 @@
+<%=form_for @facility do |f|%>
+    <div class="p-2">
+        <%=f.text_field  :name ,placeholder:"Name of Facility"%>
+    </div>
```

Comment:
To add an HTML class to a Rails form, you can do it in a more Rails-y way with cleaner syntax (allowing you to remove all of the `<div>`s):

`<%= f.text_field :name ,placeholder: "Name of Facility", class: 'p-2' %>`